TombEngine/Objects/Effects/Boss.cpp

```diff
@@ -217,7 +217,7 @@ namespace TEN::Effects::Boss
 					item.Pose.Position.x + (Random::GenerateInt(0, 127) - 64 * 2),
 					(item.Pose.Position.y - CLICK(2)) + (Random::GenerateInt(0, 127) - 64 * 2),
 					item.Pose.Position.z + (Random::GenerateInt(0, 127) - 64 * 2),
-					2, -2, 2, item.RoomNumber);
+					3 - greenExplosion, -2, greenExplosion * 2, item.RoomNumber);
```

Comment:
What the hell is happening? `greenExplosion` is a bool variable!!!

Reply:
Fixed in develop branch.